docs/publishers/embedding.rst

```diff
@@ -44,3 +44,30 @@ document and have the same origin.
 
 The client will watch for new iframes being added to the document and will
 automatically enable annotation for them.
+
+
+Clipboard permissions when loaded in an iframe
+----------------------------------------------
+
+There is a few places in the sidebar where the user can copy content to the
+clipboard:
+
+* The link, when sharing an annotation.
+* The sidebar's version.
+* The content, when exporting annotations to any format.
+* Etc.
+
+This is done through the browser's
+`Clipboard API <https://developer.mozilla.org/en-US/docs/Web/API/Clipboard_API>`_,
+which requires allowing the ``clipboard-write`` permission in Chrome (but not
+Safari or Firefox).
+
+When loading the sidebar in the top-level document, this should work out of
+the box, but if you load Hypothesis inside an iframe, remember adding the
+``allow`` attribute with the right permissions.
+
+.. code-block:: html
+
+   <iframe allow="clipboard-write">
+     <!-- Hypothesis is loaded here -->
+   </iframe>
```

Comment:
We should say something about what happens if the permissions are unavailable. eg. "If these permissions are not available, the corresponding functionality in Hypothesis will either be unavailable or will fail with an error when used."